app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -1 +1,25 @@
 /* Fork specific bootstrap overrides go here. This is included before bootstrap. */
+
+/* 979px - 980px is where nav-collapse gets triggered */
+@media (max-width: 979px) {
```

Comment:
This seems like the wrong file for this. This file is for fork-specific changes like defining NU's purple. It's one of the files that should rarely change in open as it could cause merge conflicts downstream. This responsive navigation should probably be global.